index.html

```diff
@@ -4853,6 +4853,67 @@ <h3>Issuer</h3>
         </p>
       </section>
 
+      <section class="informative">
+        <h4>Holder</h4>
+        <p>
+The value associated with the <code>holder</code> <a>property</a> is expected
+to identify the <a>holder</a> to the <a>verifier</a>.
+        </p>
+        <p>
+Relevant metadata about the <code>holder</code> <a>property</a> is expected
+to be available to the <a>verifier</a>. For example, a <a>holder</a> can
+publish information containing the verification material used to secure
+<a>verifiable presentations</a>. This metadata is relevant when
+checking proofs on <a>verifiable presentations</a>.
```

Comment:
This section presumes far too much on behalf of the verifier.

VCs are most useful when, in fact, the verifier knows nothing about holder except what is presented by VCs and VPs.

It may be that you are thinking in terms of a specific securing mechanism, but to be clear, if I have a DID from a method I'm willing to accept, there is no meta-data required to verify. The DID provides the data I need.

Reply:
> if I have a DID from a method I'm willing to accept, there is no meta-data required to verify. The DID provides the data I need.

...which implies that there is no *relevant metadata* to be available, which satisfies the first sentence.

What would you change?

Reply:
If there is no metadata, then I would not expect it to be available. 

Perhaps more importantly, this notion that a holder is going to publish anything is confusing. Holders don't "publish".

I think this is *trying* to explain that if the holder property is not a DID, you can do magic, but if it's not, you need a bunch of extra metadata. But its trying to say this without mentioning DIDs.

However, this is not correct. It could be a simple URL with a type someone knows how to use to do the verification.

Reply:
This is copy pasted from the issuer section on security... IMO it either applies to issuers and holders consistently (meta data related to keys is required to check proofs), or holders don't have keys / presentations don't have security.

Reply:
> [@jandrieu] if the holder property is not a DID, you can do magic, but if it's not, you need a bunch of extra metadata

I'm confused by `if ... not, but if ... not`. It seems that one of these should not have the `not`.

> [@OR13] This is copy pasted from the issuer section on security

Why is anything being said identically in two sections?

> [@OR13] applies to issuers and holders consistently

I think issuers (who will be issuing _MANY_ VCs) are expected to have substantially more VC-related infrastructure than holders (who will be issuing relatively small numbers of VCs/VPs, probably via wallets in most if not all cases), so I don't think much will `[apply] consistently` between them. I'm not yet convinced that this should be one of those things that `[apply] consistently`.